app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
```

Comment:
- [ ] **OPTIONAL**
Please consider using the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) `after_save` in order to call the `update_comment` method every time a new comment has been saved, then make the method private